packages/typescript/test/ts_project/bridge_js_library/tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+  "compilerOptions": {
+    "declaration": true,
+    "types": [],
+    // Help TypeScript locate the a.d.ts file from dep.
+    // Note that it comes from a js_library which means the .d.ts file is copied to the bazel-out folder.
+    // Needed when running in a sandbox or remote.
+    "rootDirs": [
+      ".",
+      "../../../../../bazel-out/darwin-fastbuild/bin/packages/typescript/test/ts_project/bridge_js_library",
```

Comment:
do we really need to get into this composite ts_project behavior to test the one-line change? I imagine there's a much simpler test that just involves
- add one .d.ts file
- expose it with a js_library
- pass that through another js_library so it should come via transitive
- assert that you get the original .d.ts provided in the DeclarationInfo (using a tiny testing rule to extract them I think)

Reply:
Sure, I'll try to get rid of the `ts_project`. Is there a place where testing rules usually live?

Reply:
it can just live right next to the test files

Reply:
Updated :)